Resources/config/services.yaml

```diff
@@ -35,3 +35,9 @@ services:
         arguments:
             $objectManager: '@doctrine.orm.entity_manager'
             $class: ~
+
+    Kontrolgruppen\CoreBundle\CPR\FaellesSQLCprService:
+        arguments:
+            $serviceUrl: '%env(CPR_SERVICE_URL)%'
```

Comment:
Consider using https://symfony.com/blog/new-in-symfony-3-4-local-service-binding to get rid of this custom service definition, e.g.
```
  bind:
    $cprServiceUrl: …
```

```
    public function __construct(HttpClientInterface $httpClient, $cprServiceUrl) …
```